python/cuml/test/test_hdbscan.py

```diff
@@ -159,7 +159,6 @@ def test_hdbscan_blobs(nrows, ncols, nclusters,
 
     cuml_agg = HDBSCAN(verbose=logger.level_info,
                        allow_single_cluster=allow_single_cluster,
-                       n_neighbors=min_samples+1,
```

Comment:
In general this but seems like it would be best to be plotted to visually see if it doesn't happen, which leads me to an idea I had some time ago: adding tests that plot things (optionally based on pytest markers) and then adding a release checklist item that is checking those tests? What do you think? (probably follow up work, but since it is relevant for this issue wanted to see your opinion)